src/App.jsx

```diff
@@ -1,33 +1,23 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { Routes, Route } from 'react-router-dom'
 import './App.css'
+import Logo from './components/Logo'
+import Header from './components/Header'
+import Navbar from './components/Navbar'
+import Articles from './components/Articles'
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+    <Logo/>
+    <Header/>
+    <Navbar/>
+    <div id="content">
+    <Routes>
+    <Route path ="/articles" element={<Articles/>}/>
+    <Route path ="/articles/:topic" element={<Articles/>}/>
```

Comment:
If you're going to have a single article page later (/articles/:article_id), then this endpoint will cause problem. It won't recognise the parametric endpoint (topic or  article_id)  e.g.  /articles/4 may get you 404 topic not found etc.